examples/layer-browser/index.html

```diff
@@ -7,6 +7,7 @@
     <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v1.13.0/mapbox-gl.css' rel='stylesheet' />
   </head>
   <body>
+    <script type="text/javascript" src="https://spectorcdn.babylonjs.com/spector.bundle.js"></script>
```

Comment:
Can we make the luma Spector support work instead of doing this? Or at least keep this change out of the PR?